docs/dc-apps-performance-toolkit-user-guide-jira.md

```diff
@@ -62,7 +62,7 @@ All important parameters are listed and described in this section. For all other
 | Parameter | Recommended value |
 | --------- | ----------------- |
 | Jira Product | Software |
-| Jira Version | The Data Center App Performance Toolkit officially supports `8.0.3` or `7.13.15` ([Long Term Support release](https://confluence.atlassian.com/enterprise/atlassian-enterprise-releases-948227420.html)) or `8.5.9` |
+| Jira Version | The Data Center App Performance Toolkit officially supports `8.0.3` and `8.5.9` or `8.13.0` or `7.13.15` ([Long Term Support release](https://confluence.atlassian.com/enterprise/atlassian-enterprise-releases-948227420.html)) |
```

Comment:
Why are long term releases not in ascending order? Could you place `7.13.15` first on the list to fix that?

Reply:
@amalyshkin thank you for your input. What do you think about having a descending order? 
I think this would be the best choice, as the latest LTS is a preferable option for most of the major clients. @mminns could you please fix the order across all User Guides to make them consistent and in descending for all Platform and LTS releases. Also, put LTS releasse first, than platform.
e.g. here line would be:
`| Jira Version | The Data Center App Performance Toolkit officially supports 
 `8.13.0`, `8.5.9` , `7.13.15` ([Long Term Support release](https://confluence.atlassian.com/enterprise/atlassian-enterprise-releases-948227420.html)) and `8.0.3` Platform Release |

Reply:
No problem, I'll do it ASAP!

Reply:
@ometelytsia, thank you. I like your suggestion. So I agree with both the descending order that you suggested and the ascending one that has been implemented by @mmizin. So I'm leaving the choice up to you.